src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,577 @@
+# Non Fungible Tokens (NFTs)
+
+The AtomicDEX API supports [ERC1155](https://www.nftstandards.wtf/Standards/ERC1155+Multi+token) and [ERC721](https://www.nftstandards.wtf/Standards/ERC721+Non+Fungible+Standard) NFTs via the [Moralis API](https://docs.moralis.io/) on the Avalanche (AVAX), BNB Smart Chain (BNB), Ethereum (ETH), Fantom (FTM), Polygon (MATIC) networks.
+
+
+
+## NFT Information Methods
+
+- Get a list of your tokens with [get_nft_list](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-list)
+- Get a list of token transfers with [get_nft_transfers](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-transfers)
+- Get token metadata with [get_nft_metadata](/atomicdex/api/v20-dev/non-fungible-tokens/get-nft-metadata)
+
+
+## NFT Transaction Methods
+
+- Withdraw ERC721 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc721)
+- Withdraw ERC1155 tokens with [withdraw_nft](/atomicdex/api/v20-dev/non-fungible-tokens/withdraw-nft#erc1155)
+
+
+## Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs. Chains must be [activated](/atomicdex/api/legacy/coin_activation) first. |
```

Comment:
User doesnt have to activate platform coin to see their NFT list, tx history or nft metadata.
Right now it is required only for withdraw nft  methods.

Reply:
removed, thanks.

Reply:
please remove `Chains must be activated` from [get tx history](https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42/files#diff-952348f4452c003b588224faac88b718b4b5a1d7c957b43a8ec35c44178d44bdR168) too

and add info that user needs to activate platform coin in [nft withdraw section](https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42/files#diff-952348f4452c003b588224faac88b718b4b5a1d7c957b43a8ec35c44178d44bdR373)

Reply:
Removed from all tables; added a note for withdraw method.